tiiuae/qgroundcontrol
# Update CMakeLists to support cmake build for UTMSP

Update CMakeLists to support cmake build for UTMSP

Description
-----------
This PR aims to improve the UTMSP library's build process by updating the CMake and Make build configurations for UTMSP support. It involves configuring required flags, linking the needed libraries, and verifying compatibility.

Test Steps
-----------
To enable and build QGC with UTMSP Adapter support, execute these commands:
1. Create a new build directory: 
```
mkdir build
```
 2. Navigate to the build directory: 
 ```
cd build
```
3. Run cmake with UTM Adapter configuration:
```
cmake -DQGC_CONFIG_UTM_ADAPTER=ON ..
```
4. Compile the build: 
```
make -j8
```

Checklist:
----------


Related Issue
-----------



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
